TeamCharter.md

```
@@ -1,102 +1,117 @@
-- відповідальність за підтримку README.md з актуальними посиланнями - @Victoria7778
-
-## 9. Індивідуальна відповідальність 
-- Кожен учасник щотижня записує відео в Loom, де показує свій внесок в роботу, а також коротко розказує, що зробила команда загалом і як саме його внесок допоміг цьому
+# Team Charter
+
+## 1. Basic Information
+
+* Team name: bottleneck
+* Members:
+
+  #### Dvoylenko Anastasia Ivanivna (GitHub: @anastasiaaq, NaUKMA email: [a.dvoilenko@ukma.edu.ua](mailto:a.dvoilenko@ukma.edu.ua))
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)